.prettierignore

```diff
@@ -4,6 +4,12 @@ packages/npm/src/*.hbs
 # Theo generated files
 packages/npm/src/tokens.scss
 packages/npm/src/theme-*.scss
+packages/npm/src/definitions/border-radius.scss
+packages/npm/src/definitions/borderRadius.ts
+packages/npm/src/definitions/color-palette.scss
+packages/npm/src/definitions/colorPalette.ts
+packages/npm/src/definitions/theme-colors.scss
+packages/npm/src/definitions/themeColors.ts
```

Comment:
💅🏻  Couldn't this be simplified?

```suggestion
packages/npm/src/definitions/*.scss
packages/npm/src/definitions/*.ts
```

Reply:
No, because I have index files there.

I place them also to avoid having to create folders during build.